tests/test_switch.py

```diff
@@ -75,6 +96,8 @@ def test_switch_attribute(self, dvs, testlog):
 
         vxlan_switch_test(dvs, switch_oid, "56789", "00:0A:0B:0C:0D:0E", "15", "56789")
 
+        ecmp_lag_hash_offset_test(dvs, switch_oid, "10", "10")
```

Comment:
Looks like we've a coverage issue. Can you please check to resolve this?

Reply:
Fixed now